src/test/java/unittest/testcases/math/MathsTests.java

```diff
@@ -118,4 +118,10 @@ public void checkFloatEquals() {
         assertFalse(Maths.floatEquals((float) a, a4), "The two floats should not be equal -- their difference is more than 0.000001f.");
         assertFalse(Maths.floatEquals(a2, a3), "The two floats should not be equal -- their difference is equal to 0.000001f.");
     }
+
+    @Test
+    public void checkLerpValues() {
+        assertEquals(5, Maths.lerp(5, 2, 0), "The lerped value should be 5.");
```

Comment:
I did forget one specific part here: Numbers should contain signifiers that show they are floating-point values.
Essentially, these numbers should have `f` at their ends.